web-app/src/main/webapp/script/loadData.js

```diff
@@ -37,7 +38,16 @@ export async function loadPoliceReports(map) {
     FILE_NAMES.map(async (file_name) => {
       const reports = await fetchAndParseJson("../data/" + file_name + ".json");
       const filteredReports = filterReports(reports, uncheckedCategories, numberOfMonths);
-      return createMarkers(map, filteredReports);
+      const markers = createMarkers(map, filteredReports);
+      for (const marker of markers) {
+        google.maps.event.addListener(marker, "click", () => {
+          const contentString =
+            `<div id="info-window"><p>${marker.crimeType}</p><p>${marker.date}</p></div>`;
+          mapComponents.infoWindow.setContent(contentString);
+          mapComponents.infoWindow.open(map, marker);
+        });
+      }
+      return markers;
```

Comment:
Can you have more than one info window at a time? The user-generated ones are set to close automatically if a new one opens. We might need to do some refactoring to make sure both functions have access to UIState.